.zuul.yaml

```diff
@@ -1,6 +1,20 @@
 ---
-- project:
-    name: infrawatch/feature-verification-tests
+- job:
+    name: feature-verification-tests-osp18
+    parent: telemetry-operator-multinode-autoscaling
+    description: |
+      Run the stf functional tests on osp18
+    run: stf_functional_tests.yml
+
+- project-template:
+    name: feature-verification-tests-osp18-template
     github-check:
       jobs:
-        - noop
+        - feature-verification-tests-osp18
+
+- project:
+    name: infrawatch/feature-verification-tests
+    templates:
+      - feature-verification-tests-osp18-template
+
```

Comment:
> Unable to freeze job graph: Job feature-verification-tests-osp18 depends on openstack-k8s-operators-content-provider which was not run.

The content provider is needed, usually it is run with the `podified-multinode-edpm-pipeline` template.
Add openstack-k8s-operators-content-provider to the project or to the template.

Probably better adding it to the template.